arches/app/models/migrations/7787_relational_data_model.py

```diff
@@ -9,7 +9,7 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.RunSQL(
-            """
+            r"""
```

Comment:
? . If this isn't a mistake I assume it's not kosher to change a migration from a previous release?

Reply:
This resolves the SyntaxWarning on python 3.12+. Eventually it will get elevated to an error, so we have to do something eventually. [more than anyone wants to know](https://adamj.eu/tech/2022/11/04/why-does-python-deprecationwarning-invalid-escape-sequence/)

Reply:
oh huh, interesting!